src/current/v25.3/query-behavior-troubleshooting.md

```diff
@@ -109,6 +109,41 @@ docker run -d --name jaeger \
 -p 6831:6831/udp -p 16686:16686 jaegertracing/all-in-one:latest
 ~~~
 
+### Log traces for transactions
+
+CockroachDB allows you to trace [transactions]({% link {{ page.version.version }}/transactions.md %}) to help troubleshoot performance issues. [Tracing]({% link {{ page.version.version }}/show-trace.md %}#trace-description) is controlled through two cluster settings that govern when a transaction trace is captured and emitted.
+
+#### Trace sampling and emission
+
+To enable tracing for a subset of transactions and emit relevant traces to the [`SQL_EXEC` logging channel]({% link {{ page.version.version }}/logging-overview.md %}#logging-channels), configure the following cluster settings:
+
+- {% include_cached new-in.html version="v25.3.0" %}[`sql.trace.txn.sample_rate`]({% link {{ page.version.version }}/cluster-settings.md %}#setting-sql-trace-txn-sample-rate): Specifies the probability (between `0.0` and `1.0`) that a given transaction will have tracing enabled. A value of `0.01` means that approximately 1% of transactions are traced. The default is `1`, which means 100% of transactions are sampled.
+- [`sql.trace.txn.enable_threshold`]({% link {{ page.version.version }}/cluster-settings.md %}#setting-sql-trace-txn-enable-threshold): Specifies a duration threshold. A trace is emitted only if a sampled transaction's execution time exceeds this value. When set to `0` (default), tracing is disabled regardless of whether the value of `sql.trace.txn.sample_rate` is greater than `0`.
+
+To emit a trace to the logs, the following conditions must be met:
+
+1. The transaction is selected based on the sampling probability.
+1. Its execution duration exceeds the configured threshold.
+
+This approach minimizes overhead by tracing a fraction of the workload and emitting traces only for potentially relevant transactions.
+
+#### Configuration example
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+-- Enable trace sampling at 1%
+SET CLUSTER SETTING sql.trace.txn.sample_rate = 0.01;
+
+-- Emit traces for sampled transactions that exceed 1s
+SET CLUSTER SETTING sql.trace.txn.enable_threshold = '1s';
+~~~
+
+With this configuration, approximately 1% of transactions are traced, and only those running longer than 1s will have their traces written to the logs. In the `SQL_EXEC` log, a line similar to the following precedes the trace:
```

Comment:
Suggest making "`SQL_EXEC` log" a link to those docs, since it's the first mention of this log channel in this subheading